engine/src/main/java/com/datatorrent/stram/StramLocalCluster.java

```diff
@@ -318,6 +333,21 @@ public StramLocalCluster(LogicalPlan dag) throws IOException, ClassNotFoundExcep
     }
   }
 
+  private void addExtraJarsToClasspath(Set<String> jars)
+      throws InvocationTargetException, IllegalAccessException, MalformedURLException, NoSuchMethodException
+  {
+    List<URL> jarUrls = new LinkedList<>();
+    for (String jarPath : jars) {
+      File file = new File(jarPath);
+      URL url = file.toURI().toURL();
+      jarUrls.add(url);
+    }
+
+    ClassLoader prevCl = Thread.currentThread().getContextClassLoader();
+    ClassLoader urlCl = URLClassLoader.newInstance(jarUrls.toArray(new URL[jarUrls.size()]), prevCl);
+    Thread.currentThread().setContextClassLoader(urlCl);
```

Comment:
Please provide unit test(s) that exercise adding custom jars to classpath and loads custom classes using both Class.forName() and context ClassLoader. As long as context ClassLoader is used to load all custom classes it should work, but I don't see how it will for with the current ClassLoader if operators or modules use Class.forName().